design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,352 @@
+    nodeRoleRef:
+      name: somenoderole
+    scalingConfig:
+      maxSize: 5
+      minSize: 1
+    updateConfig:
+      maxUnavailablePercentage: 50
+      force: true
+  providerConfigRef:
+    name: example
+```
+
+This would allow users to specify fields that would only be used on creation,
+but not on updates.
+
+When it comes to the implementation, the handling would all need to be done
+on the provider level. The `initProvider` fields should only be used in the
+`Create` step of the managed resource reconciliation loop.
+
+Support for `initProvider` would need to be a manual change in each provider.
+From our side we could provide a helper function to merge the `initProvider`
+and `forProvider` fields which would be used in the `Create` step.
+
+### Feature Gating
+
+The new management policy would use the same feature gate as ObserveOnly. So
+it would be an alpha feature that will be off by default and will be
+controlled by `--enable-alpha-management-policies` flag in Providers.
+
+The `initProvider` field would require a separate feature gate. So it would
+also be disabled by default and be controlled by `--enable-alpha-init-provider`
+flag in the Provider.
+
+### Examples how the new functionality would affect the use cases
+
+Some use cases mentioned in the beginning of the document would be
+solved just by setting the new management policy to `PartialControl`. For
+others, the `initProvider` field would need to be used, mostly because
+the fields in question are required on creation.
+
+#### AutoScalingGroup
+
+The `desiredCapacity` is not required on creation, but populated by late
+initialization. It can be solved just using the `PartialControl` management
+policy.
+
+#### DynamoDB Table
+
+The `readCapacity` and `writeCapacity` are not required, but they are required
+if the `billingMode` is set to `PROVISIONED`. So the `initProvider` field
+would need to be used alongside `PartialControl` management policy.
+
+#### EKS NodeGroup
+
+The `scalingConfig.desiredSize` is required so `initProvider` would need
+to be used alongside `PartialControl` management policy.
+
+#### GCP Node Pool
+
+Just using the `PartialControl` management policy would be enough as the
+`version` is not required on creation.
+
+#### Azure AD Group
+
+The use case where `members` and `owners` want to be set on creation and
+then ignored on updates would be solved by using `initProvider` alongside
+`PartialControl` management policy.
+
+#### Tags
+
+Setting initial tags should work with `initProvider`. However, it seems that
+the providers are setting some default tags in the into the `forProvider`
+field in the `Initialize` step and are updating the resource, so this would
+need to be changed to use `initProvider` instead, or we can check if those
+tags are actually needed or just skip setting them similarly to how
+ObserveOnly does it.
+
+Ref: [Upjet Initialize] or [AWS community provider tag example].
+
+## Solution alternative considered
```

Comment:
Could you also add some notes on why we didn't go with the alternative solutions?